src/PKSim.R/Services/IndividualFactory.cs

```diff
@@ -0,0 +1,87 @@
+﻿using System;
+using System.Linq;
+using OSPSuite.Core.Domain;
+using OSPSuite.Core.Domain.Populations;
+using OSPSuite.Core.Domain.Services;
+using PKSim.Core.Mappers;
+using PKSim.Core.Model;
+using PKSim.Core.Model.Extensions;
+using PKSim.Core.Snapshots.Mappers;
+using PKSim.R.Domain;
+using ICoreIndividualFactory = PKSim.Core.Model.IIndividualFactory;
+using OriginData = PKSim.Core.Snapshots.OriginData;
+
+namespace PKSim.R.Services
+{
+   public interface IIndividualFactory
+   {
+      /// <summary>
+      /// Returns the <seealso cref="ParameterValues"/> representing an individual with the characteristics defined in <paramref name="individualCharacteristics"/>.
+      /// </summary>
+      ParameterValue[] CreateIndividual(IndividualCharacteristics individualCharacteristics);
```

Comment:
@PavelBal This is where I need your input

@Yuri05 please comment
Any idea why we have two methods here? Shouldn/t create individual only return distributed parameter anyways?

Right now it returns everyparameter that is changed either directly or indirectly by create Individual .But then it returns also formula paraemter which we then try to filter out etc... Shouldn't we not do that at the first place and only return distributed parameters?

Reply:
> Any idea why we have two methods here? Shouldn/t create individual only return distributed parameter anyways?

I agree... I do not see a point in returning (explicit) formula parameters.